src/browser/modules/Editor/language/consoleCommands.js

```diff
@@ -0,0 +1,123 @@
+export default [
+  {
+    name: ':help',
+    description: ' - learn about other topics',
+    commands: [
+      { name: 'commands' },
+      { name: 'play' },
+      { name: 'server' },
+      { name: 'cypher' },
+      { name: 'param' },
+      { name: 'params' },
+      { name: 'queries' },
+      { name: 'keys' },
+      { name: 'clear' },
+      { name: 'server-user' },
+      { name: 'match' },
+      { name: 'where' },
+      { name: 'return' },
+      { name: 'create' },
+      { name: 'merge' },
+      { name: 'delete' },
+      { name: 'detach-delete' },
+      { name: 'set' },
+      { name: 'foreach' },
+      { name: 'with' },
+      { name: 'load csv' },
+      { name: 'unwind' },
+      { name: 'start' },
+      { name: 'create-unique' },
+      { name: 'create-index-on' },
+      { name: 'starts-with' },
+      { name: 'ends-with' },
+      { name: 'contains' },
+      { name: 'rest' },
+      { name: 'rest-get'},
+      { name: 'rest-put'},
+      { name: 'rest-delete'},
+      { name: 'rest-post'}
+    ]
+  },
+  {
+    name: ':play',
+    description: ' - loads a mini-deck with either guide material or sample data',
+    commands: [
+      { name: 'intro' },
```

Comment:
There are loads more guides than this list, but I'm sure we can the missing ones in later